README.md

```diff
@@ -78,27 +78,32 @@ However, many IDEs can be configured to auto format on save.
 
 ## Proposed Reimbursement Work Flow
 
-The proposed workflow is to be automated and run on a schedule of once per _accounting
-period_ at N hours past the end of the accounting period. Note that we must wait some
-time after the period has ended for some data to finalize (e.g. `dex.trades`
-, `ethereum.transactions` our event data, etc...).
+The solver reimbursements are to be executed each Tuesday with the accounting period of the last 7 days. If the payout can not be done on Tuesdays, its okay to execute them later, but still the accounting period should always be from 00:00 between Monday and Tuesday to 00:00 between Monday and Tuesday
 
-1. Run the Reimbursement & Rewards Query
+In order to do the payout, run the following scripts:
+```
+source .env
+rm -r out/
+python -m src.fetch.transfer_file --start '2022-MM-DD' --end '2022-MM-DD'
```

Comment:
We should not have to specify both dates. We should adapt the script to accept either

- an integer (the calendar  week) or
- the start date a date string `YYYY-MM-DD`.

The rest should be handled internally. I will make a PR for this.

Reply:
Yes. I was thinking the same but wanted to proceed with it right now for the first payout.
If you can create the PR quickly that would be great